fuel-tx/src/transaction/consensus_parameters.rs

```diff
@@ -320,50 +467,156 @@ pub struct PredicateParameters {
     pub max_gas_per_predicate: u64,
 }
 
-impl PredicateParameters {
-    /// Default consensus parameters with settings suggested in fuel-specs
+impl PredicateParametersV1 {
+    /// Default parameters just for testing.
```

Comment:
If these are just for testing, shouldn't we feature-gate them?

Reply:
I will put them under `test-helpers` in a separate PR